src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -131,24 +132,25 @@ public void Dispose()
     }
 
     [SupressImportIntoVM]
-    [Obsolete("Do not use! This will be subject to changes in a future version of Dynamo")]
+    [Obsolete("Deprecated. Please use Dynamo.PythonServices.EvaluationState instead.")]
     public enum EvaluationState { Begin, Success, Failed }
 
     [SupressImportIntoVM]
-    [Obsolete("Do not use! This will be subject to changes in a future version of Dynamo")]
+    [Obsolete("Deprecated. Please use evaluation handlers from Dynamo.PythonServices instead.")]
     public delegate void EvaluationEventHandler(EvaluationState state, PyScope scope, string code, IList bindingValues);
 
     /// <summary>
     ///     Evaluates a Python script in the Dynamo context.
     /// </summary>
     [IsVisibleInDynamoLibrary(false)]
-    public static class CPythonEvaluator
+    public class CPythonEvaluator : Dynamo.PythonServices.PythonEngine
     {
         private const string DynamoSkipAttributeName = "__dynamoskipconversion__";
         private const string DynamoPrintFuncName = "__dynamoprint__";
         private const string NodeName = "__dynamonodename__";
-        static PyScope globalScope;
         internal static readonly string globalScopeName = "global";
+
+        private static PyScope globalScope;
         private static DynamoLogger dynamoLogger;
         internal static DynamoLogger DynamoLogger {
```

Comment:
So these also need not be static anymore, do they?

Reply:
They can still be static..or converted to singleton members
Should be no real difference for now...
I chose to leave them as they were to avoid too many changes